tests/unit_tests/python/test_validators.py

```diff
@@ -2,13 +2,43 @@
 
 import jsonschema.exceptions
 import pytest
+import fnmatch, json, os, os.path, re
 
-from api import validators
+from api import config, validators
 
 log = logging.getLogger(__name__)
 sh = logging.StreamHandler()
 log.addHandler(sh)
 
+# Enable to force failure if example data is missing
+FAIL_ON_MISSING_EXAMPLE = True
+
+SCHEMAS_PATH = config.schema_path + '/'
+EXAMPLES_PATH = os.path.join(SCHEMAS_PATH, '../examples')
+LIST_SCHEMA = re.compile(r'(\w+)-list')
+
+# These schemas will not fail if there is no example
+IGNORED_SCHEMAS = [
+    'input/container.json',
+    'input/enginemetadata.json', # TODO: Needs an example, is the schema really up to date?
+    'input/search.json', # TODO: Is this used?
```

Comment:
This is not used, and can get removed until it is replaced by an up-to-date schema